src/plugins/visualizations/public/embeddable/_embeddables.scss

```diff
@@ -45,4 +45,21 @@
   .visLegend__toggle {
     opacity: 1; /* 3 */
   }
+}
+
+.embPanel {
+  .visualize[data-loading],
+  .visualize[data-error] {
+    pointer-events: none;
+    -webkit-filter: grayscale(100%);
+    -moz-filter: grayscale(100%);
+    -ms-filter: grayscale(100%);
+    -o-filter: grayscale(100%);
+    filter: grayscale(100%);
+    filter: gray;
+  }
+
+  .panel-label {
```

Comment:
It probably question to @mdefazio, but would it be better to make this `position:absolute` to not cause chart shift when this pops-up?

Reply:
I wanted to do this, but was scared that someone will be angry at me for using `position: absolute` :)

Reply:
I guess this is a valid use case to have this in a left hand corner on top of the chart 👍
Let's see what design thinks

Reply:
Using absolute positioning is fine so long as you monitor any other children of the same parent where you have to add `position: relative` in order to accurately position the badge. Meaning, make sure that there aren't any other absolutely positioned children that actually rely on a different parent's relative positioning.

Also, this class is too generically named. Please remember to use BEM naming conventions with plugin prefixes.